runtime/battery-station/src/parameters.rs

```diff
@@ -183,6 +183,9 @@ parameter_types! {
     /// (Slashable) The orcale bond. Slashed in case the final outcome does not match the
     /// outcome the oracle reported.
     pub const OracleBond: Balance = 50 * CENT;
+    /// (Slashable) A bond for an outcome reporter, who is not the oracle.
+    /// Slashed in case the final outcome does not match the outcome by the outsider.
+    pub const OutsiderBond: Balance = 25 * CENT;
```

Comment:
Need your opinion on this value. Open to discuss.

Reply:
Either same as oracle bond or significantly higher. We're looking to discourage users from making a report unless they are absolutely certain they're doing the right thing.

Reply:
2399ceb0b3fd594671ffe4eb320429cdb1c3fda6